### PR TITLE
DSERV-65-ic-match-grch38

### PR DESCRIPTION
**Pull request by mingjiecn**

### CODE REVIEW

File: genomic_data_service/regulome_atlas.py [hunk 1]
```diff
@@ -56,38 +57,73 @@
...
+except RuntimeError:
+    IC_MATCHED_MAX_BW_GRCH38 = None
+
+try:
+    if file_IC_max_grch38_exists:
+        IC_MAX_BW_GRCH38 = pyBigWig.open(FILE_IC_MAX_GRCH38_PATH_LOCAL)
+    else:
+        IC_MAX_BW_GRCH38 = pyBigWig.open(FILE_IC_MAX_GRCH38_PATH_REMOTE)
+except RuntimeError:
+    IC_MAX_BW_GRCH38 = None
```
Comment (ottojolanki) [inline genomic_data_service/regulome_atlas.py]: Looks like we are repeating the same code here. Could be better to use a function?

File: genomic_data_service/regulome_atlas.py [hunk 2]
```diff
@@ -56,38 +57,73 @@
...
 except FileNotFoundError:
     TRAINED_REG_MODEL = None
 
-file_IC_matched_max_exists = exists(FILE_IC_MATCHED_MAX_PATH_LOCAL)
-file_IC_max_exists = exists(FILE_IC_MAX_PATH_LOCAL)
+file_IC_matched_max_hg19_exists = exists(FILE_IC_MATCHED_MAX_HG19_PATH_LOCAL)
+file_IC_max_hg19_exists = exists(FILE_IC_MAX_HG19_PATH_LOCAL)
+file_IC_matched_max_grch38_exists = exists(
+    FILE_IC_MATCHED_MAX_GRCH38_PATH_LOCAL)
+file_IC_max_grch38_exists = exists(FILE_IC_MAX_GRCH38_PATH_LOCAL)
```
Comment (ottojolanki) [inline genomic_data_service/regulome_atlas.py]: If we wrap the code below in a function, we can probably get rid of these separate existence checks.

**Review by ottojolanki — CHANGES_REQUESTED** [review]
I see an opportunity to get rid of some code repetition. Seems good otherwise.

File: genomic_data_service/regulome_atlas.py (hunk 1 above)
Comment (mingjiecn) [reply genomic_data_service/regulome_atlas.py]: Sounds good. Will work on it.

**Review by ottojolanki — APPROVED** [review]
LGTM

**Pull request merged**